static/app/views/settings/organizationMembers/organizationMemberRow.tsx

```diff
@@ -216,7 +213,7 @@ export default class OrganizationMemberRow extends PureComponent<Props, State> {
                     : isPartnershipUser
                       ? t('You cannot make changes to this partner-provisioned user.')
                       : // only show this message if member can remove invites but invite was not sent by them
-                        isInvite && canRemoveInvites && !isInviteFromCurrentUser
+                        pending && canInvite && !isInviteFromCurrentUser
                         ? t('Your role cannot modify this invite.')
```

Comment:
would it be better to say "You cannot modify this invite" bc `canInvite` is `True`?